app/managers.py

```diff
@@ -0,0 +1,43 @@
+from models import Actor
+import sqlite3
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connect = sqlite3.connect("cinema.sqlite")
```

Comment:
Consider implementing a method to close the database connection to prevent resource leaks.